chainstate/storage/src/internal/version.rs

```diff
@@ -19,7 +19,7 @@ use serialization::{Decode, Encode};
 pub struct ChainstateStorageVersion(u32);
 
 impl ChainstateStorageVersion {
-    pub const CURRENT: Self = Self(9);
+    pub const CURRENT: Self = Self(10);
```

Comment:
Why do we need this? We didn't change the storage scheme at all, right?

Reply:
I added indexes to OutputValue enum

Reply:
Just adding indexes doesn't warrant this as long as they're not change. Your change just made them explicit. In fact, if we changed them, we'd have much bigger problems.